<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>734</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1170</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1180</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1180</width>
     <height>680</height>
    </size>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Parcial FInal Digitales III&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="JAVIER">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Presentado por: Javier Alexis Cardenas y Johan Sebasti√°n Casagua&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1171</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Sensores</string>
     </attribute>
     <widget class="QFrame" name="verticalFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>571</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L1">
       <item>
        <widget class="QFrame" name="chart1">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="verticalFrame_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>551</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L2">
       <item>
        <widget class="QFrame" name="chart2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="verticalFrame_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>420</y>
        <width>551</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L3">
       <item>
        <widget class="QFrame" name="chart6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="verticalFrame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>571</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L4">
       <item>
        <widget class="QFrame" name="chart3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="verticalFrame_6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>210</y>
        <width>551</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L6">
       <item>
        <widget class="QFrame" name="chart4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="verticalFrame_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>420</y>
        <width>571</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L5">
       <item>
        <widget class="QFrame" name="chart5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Coordenadas</string>
     </attribute>
     <widget class="QFrame" name="verticalFrame_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>571</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L1_2">
       <item>
        <widget class="QFrame" name="chartx">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="verticalFrame_8">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>50</y>
        <width>571</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L1_3">
       <item>
        <widget class="QFrame" name="charty">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="verticalFrame_9">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>330</y>
        <width>571</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="L1_4">
       <item>
        <widget class="QFrame" name="chartz">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>420</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ahi estan todas las lineas... lo que pasa es que como algunas tuberias el numero aleatorio es el mismo, pues entonces las lineas se superponen &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>171</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tambien toca maximixar el programa para que los cuadros se acomoden o simplemente modificar alguno de los bordes (ampliar la ventana o reducirla)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
